src/promptflow/promptflow/executor/_process_manager.py

```diff
@@ -0,0 +1,458 @@
+import multiprocessing
+from multiprocessing import Queue
+from typing import List
+import queue
+import signal
+from functools import partial
+from dataclasses import dataclass
+from enum import Enum
+import psutil
+from promptflow._core.operation_context import OperationContext
+from promptflow._utils.logger_utils import LogContext, bulk_logger
+from promptflow.executor.flow_executor import FlowExecutor
+
+
+@dataclass
+class ProcessInfo:
+    index: int = None
```

Comment:
We may not have default values if the default None is not expected.

Reply:
fixed